Applications/Android/kotlin/app/src/main/java/com/samsung/sr/nntr/nntrainer/Trainer.kt

```diff
@@ -0,0 +1,60 @@
+package com.samsung.sr.nntr.nntrainer
+
+import android.content.Context
+import android.util.Log
+
+class NNTrainerNative {
+   init{
+      System.loadLibrary("nntrainer_resnet")
+   }
+
+   external fun nntrainerTrain(tdata: FloatArray, batch_size:Int, data_size: Int, data_len: Int, label_len: Int, dir_path: String, file_path: String, best_path:String, epoch: Int ): Int
+
+   external fun nntrainerTest(vdata: FloatArray, output: FloatArray, data_size: Int, data_len: Int, label_len: Int, dir_path:String, file_path: String, correct:Int ): Int
+}
+
+
+class Trainer(private val context: Context ) {
+
+    fun train() {
+    	val batch_size = 16
+	val epoch = 2
+
+	var model_bin_path = context.filesDir.toString()+"/weight.bin"
```

Comment:
I'm not sure, but there are some kotlin style checkers.
https://github.com/pinterest/ktlint
and, vscode plugin: https://marketplace.visualstudio.com/items?itemName=esafirm.kotlin-formatter

Applying formatter would be better in the later.

Reply:
Great. I will check if there is available kotlin style checker.